cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -101,6 +133,16 @@ metadata:
     operator: rook
     storage-backend: ceph
 rules:
+    # Most resources are represented by a string representation of their name, such as “pods”, just as it appears in the URL for the relevant API endpoint.
+    # However, some Kubernetes APIs involve a “subresource”, such as the logs for a pod. [...]
+    # To represent this in an RBAC role, use a slash to delimit the resource and subresource.
+    # https://kubernetes.io/docs/reference/access-authn-authz/rbac/#referring-to-resources
+  - apiGroups: [""]
+    resources: ["pods", "pods/log"]
+    verbs: ["get", "list"]
+  - apiGroups: [""]
+    resources: ["pods/exec"]
+    verbs: ["create"]
```

Comment:
Do these need to be here if they are present in the ClusterRole and ClusterRoleBinding?

Reply:
Good question I'll test. I was seeing a strange behavior previously.

Reply:
Not needed!